configs/nextflow.hopper.config

```diff
@@ -303,13 +303,13 @@ myeloid {
     params.mei_list = "${params.refpath}/annotation_dbs/MELT/mei_list.txt"
     params.meltheader = "${params.refpath}/annotation_dbs/MELT/melt_vcf_header"
     params.svdb = "${params.refpath}/annotation_dbs/onco-solid/artefact-sv/200513/svdb_200513.db"
-    params.svrank_model_s = "/fs1/resources/scout/rank_models_38/svrank_single_v1.0_panel__VEP111.ini"
-    params.rank_model = "/fs1/resources/scout/rank_models_38/rank_model_cmd_v4__VEP111.ini"
-    params.rank_model_s = "/fs1/resources/scout/rank_models_38/rank_model_cmd_v5.0_single_withoutmodels_onco__VEP111.ini"
-    params.svrank_model = "/fs1/resources/scout/rank_models_38/svrank_model_v1.0__VEP111.ini"
+    params.svrank_model_s = "/fs1/resources/scout/rank_models_38/myeloid_noloqusdb/svrank_single_v1.0_panel__VEP111.ini"
```

Comment:
good to keep rank models separated by subdirs, could perhaps make it easier to make sense of all the different files